.babelrc

```diff
@@ -6,6 +6,11 @@
     },
     "prod": {
       "plugins": []
+    },
+    "start-webpack-dev-server": {
```

Comment:
so when we start webpack dev server from npm scripts, then it reads into this config and sets the react-hmre preset?